Toluene/Vasudevan 2005 yaml files/Vasudevan_phi_0.5.yaml

```diff
@@ -0,0 +1,87 @@
+---
+file-author:
+    name: Morgan Mayer
+    ORCID: 0000-0001-7137-5721
+file-version: 0
+chemked-version: 0.3.0
+reference:
+    doi: 10.1016/j.proci.2004.07.039
+    authors:
+        - name: V. Vasudevan
+        - name: D. F. Davidson
+        - name: R. K. Hanson
+    journal: Proceedings of the Combustion Institute
+    year: 2005
+    volume: 30
+    pages: 1155-1163
+    detail: Vasudevan_phi_0.5
+experiment-type: ignition delay
+apparatus:
+    kind: shock tube
+    institution: High Temperature Gas dynamics Laboratory, Mechanical Engineering Department, Stanford University
+    facility: stainless steel shock tube
+common-properties:
+    composition: &comp
+      kind: mole fraction
+      species:
+        - species-name: toluene
+          InChI: 1S/C7H8/c1-7-5-3-2-4-6-7/h2-6H,1H3
+          amount:
+            - 0.001
+        - species-name: O2
+          InChI: 1S/O2/c1-2
+          amount:
+            - 0.018
+        - species-name: Ar
+          InChI: 1S/Ar
+          amount:
+            - 0.981
+    ignition-type:  &ign
+        target: OH
+        type: 1/2 max
+datapoints:
+    - temperature:
```

Comment:
Same here, uncertainty estimates are given on pg. 1156

Reply:
I haven't used uncertainties in a file before.  Is this a line that goes under each data point or in the common properties?

Reply:
Check out the bit in here: https://pr-omethe-us.github.io/PyKED/ck-tutorial.html#the-data-section